etna/lib/etna/clients/janus/models.rb

```diff
@@ -69,6 +69,18 @@ def html
         end
       end
 
+      class JsonResponse
```

Comment:
Hmm, so we're not filling out some of the attributes a bit more?
One of the advantages of having the explicit attribute expansion was catching simple bugs and attribute renaming / defaults logics in the face of the api shape changing over time.  It also meant that there was a semi documentation on what sorts of things were available consistently in the response. Using a hash, we frequently have nil checks and that's about it.

Reply:
Oh, good point! Sorry, I forgot -- I can update that.